src/components/PoolInfo/index.tsx

```diff
@@ -0,0 +1,125 @@
+import React, {
+  useCallback,
+  useState,
+  useEffect,
+  useMemo,
+  useContext,
+} from "react";
+import { Contract } from "ethers";
+import { useWeb3React } from "@web3-react/core";
+
+import { fetchFundComposition } from "../../api";
+import PoolLogicAbi from "../../contracts/PoolLogic.json";
+import PoolManagerLogicAbi from "../../contracts/PoolManagerLogic.json";
+import { fromDecimal } from "../../utils";
+import {
+  FundCompositionContext,
+  InvestModalContext,
+  PoolAddressContext,
+} from "../../context";
+
+const PoolInfo: React.FC = () => {
+  const [depositAssets, setDepositAssets] = useContext(FundCompositionContext);
+  const poolAddress = useContext(PoolAddressContext);
+  const [userBalance, setUserBalance] = useState<number>(0);
+
+  const [isPrivate, setIsPrivate] = useState<boolean>(false);
+  const [managerAddress, setManagerAddress] = useState<string>("");
+  const [privatePoolMembership, setPrivatePoolMembership] =
+    useState<boolean>(false);
+
+  const [_, setInvestModalOpen] = useContext(InvestModalContext);
```

Comment:
Why the underscore? If you don't use this value, you can remove it, leaving only the comma